### ISSUE TITLE
SI units

### ISSUE THREAD

**Issue by ali-ramadhan** [OceanBioME/OceanBioME.jl#235]
Out of curiosity, how difficult would it be to convert OceanBioME.jl to fully utilize SI units?

I know existing models and parameters aren't always formulated using SI units but I'm wondering if it will make OceanBioME.jl easier to use, especially interacting with other packages.

My favorite unit so far has been gC / dm² / s / einstein!

**Comment by jagoosw** [comment]
I don't think it would be that hard because each of the models is self consistent, but it might be quite a bit of work I think.

Also, I think all of the models use the same units except for the sugar kelp model (but the giant kelp model has more consistent units) which should be mmol X/m $^3$ which is the same as $\mu$ mol/litre, and use watts instead of einsteins for light.
